walkthroughs/week-3-server/examples/todo-list/src/main/webapp/script.js

```diff
@@ -47,6 +47,9 @@ function createTaskElement(task) {
 /** Tells the server to delete the task. */
 function deleteTask(task) {
   const params = new URLSearchParams();
+  console.log(params);
```

Comment:
Log lines like this are helpful for debugging, but remove them before submitting your PR